packages/xo-server-perf-alert/src/index.js

```diff
@@ -524,6 +524,31 @@ ${monitorBodies.join('\n')}`
                   })
                 }
 
+                const checkManagementAgent = (vm, guestMetrics) => {
+                  if ((vm.power_state !== 'Running' && vm.power_state !== 'Paused') || guestMetrics === undefined) {
+                    return false
+                  }
+
+                  const { major, minor } = guestMetrics.PV_drivers_version
+                  const hasPvVersion = major !== undefined && minor !== undefined
+                  return hasPvVersion || guestMetrics.other['feature-static-ip-setting'] === '1'
+                }
+
+                if (lcObjectType === 'vm' && definition.variableName === 'memoryUsage') {
+                  const vm = result.object
+                  const guestMetrics = this._xo.getXapi(uuid).getObject(vm.guest_metrics)
+
+                  const managementAgentDetected = checkManagementAgent(vm, guestMetrics)
+
+                  result.listItem = `  * ${result.objectLink}: ${
+                    result.value === undefined
+                      ? "**Can't read performance counters**"
+                      : !managementAgentDetected
+                        ? 'Guest tools must be installed'
+                        : result.value.toFixed(1) + typeFunction.unit
+                  }\n`
```

Comment:
I don't think this will work.
`result.listItem` will be replaced by `result.listItem = ...` at line `552`